host.go

```diff
@@ -184,12 +281,13 @@ func (h *gpbftHost) SetAlarm(at time.Time) {
 // based on the decision received (which may be in the past).
 // E.g. this might be: finalised tipset timestamp + epoch duration + stabilisation delay.
 func (h *gpbftHost) ReceiveDecision(decision *gpbft.Justification) time.Time {
-	h.log.Infof("got decision: %+v", decision)
+	h.log.Errorf("got decision, finalized head at epoch: %d", decision.Vote.Value.Head().Epoch)
```

Comment:
error?

Reply:
temporary, to distinguish it from log spam